File: Backend/src/main/java/umg/edu/analisis_ii/sistema_votaciones/models/Candidato.java
package umg.edu.analisis_ii.sistema_votaciones.models;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;
/**
 * @author mvracot
 * */
@Entity
@Table(name = "candidatos", schema = "sistema_votaciones")
@Getter
@Setter
@NoArgsConstructor
public class Candidato {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column(name = "nombre_pres", nullable = false)
    private String nombrePres;

    @Column(name = "nombre_vice")
    private String nombreVice;

    @Column(name = "partido_politico")
    private String partidoPolitico;

    @Column(name = "imagen_url")
    private String imagenUrl;

    @Column(name = "tipo_candidato", nullable = false)
    private String tipoCandidato;

    // Getters and setters
}

File: FrontEnd/Sistema-de-votaciones-frontend/src/app/models/candidato.ts
export interface Candidato {
  id: number;
  nombre_vice: string;
  imagen_url: string;
  nombre_pres: string;
  partido_politico: string;
  tipo_candidato: string;
  seleccionado: boolean;
}



export interface Voto{
  cui_votante: string;
  presidente_id: number;
  alcalde_id: number;
  diputado_lista_id: number;
  diputado_distrito_id: number;
  parlacen_id: number;
}
